templates/lp12-module/bbscaff.js

```diff
@@ -0,0 +1,42 @@
+module.exports = function(bbscaff) {
+    bbscaff.prompt([
+        {
+            name: 'module_title',
+            message: 'Module title'
+        },
+        {
+            name: 'module_name',
+            message: 'Module name',
+            'default': function(answers) {
+                return answers.module_title.replace(/ /gi, '-').toLowerCase();
+            }
+        },
+        {
+            name: 'module_author',
+            message: 'Module author',
+            'default': 'Backbase <practices@backbase.com>'
+        },
+        {
+            name: 'module_version',
+            message: 'Module version',
+            'default': '1.0.0'
```

Comment:
Following semantic versioning, it's better to start with `0.1.0`. Unless it's political question.